src/waypoint.jsx

```diff
@@ -11,6 +11,7 @@ const POSITIONS = {
 const defaultProps = {
   topOffset: '0px',
   bottomOffset: '0px',
```

Comment:
It seems like we should rename these props at some point to be `offsetBefore` and `offsetAfter`. If we do that, it would be good to keep these around with a deprecation warning for one major release.